source/core/src/main/com/csse3200/game/components/CombatStatsComponent.java

```diff
@@ -22,8 +22,8 @@ public class CombatStatsComponent extends Component {
   private final int maxExperience;
 
   public CombatStatsComponent(int health, int hunger, int strength, int defense, int speed, int experience) {
-      this.maxHealth = health;
-      this.maxHunger = hunger;
+      this.maxHealth = 100;
```

Comment:
Allowing for health to be initialised to something other than the input health (probably should add this as an input to the constructor.